tests/integration/ledger/test_ledger.py

```diff
@@ -0,0 +1,116 @@
+# Require a connected Ledger ...
+
+import pytest
+
+from hexbytes import (
+    HexBytes,
+)
+
+from eth_account import (
+    Account,
+)
+from eth_account.messages import (
+    defunct_hash_message,
+)
+from eth_account.signers.ledger import (
+    LedgerAccount,
+)
+
+# Not a fixture because it does not support multiple LedgerAccount
+ledger = LedgerAccount()
```

Comment:
What does it mean that "it does not support multiple LedgerAccount"? Maybe it should be a session fixture, like:

```py
@pytest.fixture(scope="session")
def ledger:
  return LedgerAccount()
```